components/infoColumns/index.tsx

```diff
@@ -0,0 +1,160 @@
+  ${breakpoints("margin", "", [{ 1400: "1rem 0" }])};
+`;
+const CardContainer = styled.div<{
+  bordered?: boolean;
+  flex?: string;
+  height?: string;
+  color?: string;
+  margins?: string;
+  pad?: string;
+}>`
+  display: flex;
+  flex: ${(props) => (props.flex ? props.flex : "1")};
+  ${breakpoints("flex", "", [{ 1400: "1" }])};
+  ${breakpoints("min-width", "", [{ 1400: "45vw" }])};
+  ${breakpoints("width", "", [{ 1000: "94vw" }])};
+  height: ${(props) => (props.height ? props.height : "auto")};
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  background-color: ${(props) => (props.color ? props.color : "transparent")};
+  margin: ${(props) => (props.margins ? props.margins : "1rem")};
+  padding: ${(props) => (props.pad ? props.pad : "1.5rem")};
+  ${breakpoints("padding", "", [{ 1400: "1.5rem auto" }])};
+  ${breakpoints("margin", "", [{ 1400: "1rem 0" }])};
+`;
+const Title = styled(SubTitle)`
+  font-size: 4rem;
+  ${breakpoints("font-size", "", [{ 1000: "2.5rem" }])};
+  width: 100px;
+  ${breakpoints("width", "", [{ 1000: "100%" }])};
+  margin: 0;
+`;
+
+const InfoColumns = () => {
+  return (
+    <InfoColumnWrapper>
+      <CardContainer flex="1" margins="1rem 0" pad="1rem 1rem 1rem 0">
+        <Title>About CIMUN</Title>
+        <Body size="1rem" line={1.4}>
+          For 18 years, CIMUN has been a premier conference for scholastic Model
+          United Nations programs. <br />
+          <br />
+          Beginning with our opening ceremonies, delegates are introduced to a
+          24/7, fully-integrated simulation, where they are challenged to work
+          across committees as they confront complex present-day and historical
+          issues. <br />
+          <br />
+          The CIMUN Philosophy believes that educational value is defined by
+          realism. In this regard, action taken in one committee affects the
+          proceedings in every other committee. Crises occur in real-time
+          throughout the conference, and CIMUN staff diligently work to create a
+          conference that is fast-paced, nuanced, and above all else,
+          educational. CIMUN recognizes that nations cannot always solve
+          problems together, and that the complexity of international affairs is
+          high. Delegates are rewarded for demonstrating comprehensive knowledge
+          of their nation's stance and role on the global stage, rather than for
+          passing empty resolutions or for compromising their nation's values.
+          <br />
+          <br />
+          As an independent conference, CIMUN is not affiliated with any
+          university. Rather, we are a diverse and talented team of students,
+          post-graduates, and professionals from backgrounds in academia,
+          finance, politics, business, technology and more to deliver a
+          high-caliber educational experience to our delegates.
+        </Body>
```

Comment:
I don't really mind this, but I personally think large blocks of text should be mapped onto an object in another file for convenient editing